src/Feature/SCM/Feature_SCM.rei

```diff
@@ -60,10 +65,17 @@ module Effects: {
 [@deriving show]
 type msg;
 
-let update: (msg, model) => (model, Isolinear.Effect.t(msg));
+module Msg: {let keyPressed: string => msg;};
+
+type outmsg =
+  | Effect(Isolinear.Effect.t(msg))
+  | Focus
+  | Nothing;
+
+let update: (ExtHostClient.t, model, msg) => (model, outmsg);
 
 let handleExtensionMessage:
-  (~dispatch: msg => unit, Oni_Extensions.SCM.msg) => unit;
+  (~dispatch: msg => unit, ExtHostClient.SCM.msg) => unit;
```

Comment:
Moved the `SCM` module into `ExtHostClient`, as it fit better there and allows `ExtHostClient.t` to be abstract again.